core/modules/qproc/ChunkSpec.cc

```diff
@@ -24,7 +24,7 @@
 /**
```

Comment:
Wait, what? Which file is missing the copyright? (but yes, ChunkSpec.cc needed to be patched for AURA/LSST instead of LSST Corp)